internals/daemon/response.go

```diff
@@ -169,22 +168,30 @@ func (f fileResponse) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	http.ServeFile(w, r, string(f))
 }
 
+// Responsef builds an error Response that returns the status and formatted message.
```

Comment:
Nit: This name needs to match the function name.

Reply:
Need to fix.

Reply:
Fixed.